### ISSUE TITLE
Error rendering

### ISSUE THREAD

**Issue by sternj** [emacs-jupyter/jupyter#476]
I'm not seeing formatting characters render when running jupyter-python (example below), is this a misconfiguration on my part?

```
#+RESULTS:
: 21.9 s ± 3.91 s per loop (mean ± std. dev. of 7 runs, 1 loop each)


#+begin_src jupyter-python :session asdf :async true
oiehdf
#+end_src

#+RESULTS:
:RESULTS:
# [goto error]
: [0;31m---------------------------------------------------------------------------[0m
: [0;31mNameError[0m                                 Traceback (most recent call last)
: Cell [0;32mIn[1], line 1[0m
: [0;32m----> 1[0m [43moiehdf[49m
: 
: [0;31mNameError[0m: name 'oiehdf' is not defined
:END:
```

Note that the Unicode characters that don't seem to be rendering in the web UI are represented as `^[` in emacs

**Comment by akirakyle** [comment]
See #380. It sounds like you may be using an older version that doesn't have this fixed.

**Comment by sternj** [comment]
Okay so I'm using doom, I unpinned `jupyter`, I ran `doom sync -u`, and the thing in `.local/straight/repos/jupyter` is up to date. However, I am running into the same issue (it's in the example block, similar to the last comment on that issue)

**Comment by akirakyle** [comment]
Alright, I can actually reproduce the issue now with the latest master. However, the issue goes away as soon as a I do `revert-buffer` and doesn't ever come back. I don't have time to do more debugging right now, but hopefully this will help narrow down the new bug.

**Comment by danielpopescu** [comment]
I have the same issue but only on Linux (and with zmq disabled). On Macos it seems to work fine.
And, indeed,  revert-buffer works! But one has to remember to do so early, before making any (or few) changes...

**Comment by jamieforth** [comment]
I came across this trying to figure out how to gracefully switch between sessions in different virtual environment (#487).

It seems to be related to the order in which the functions `org-babel-jupyter-make-local-aliases` and `jupyter-org-interaction-mode` get called in `org-mode-hook`. Making sure that `jupyter-org-interaction-mode` is called first solves the problem for me.

It's only a tiny change to the code https://github.com/emacs-jupyter/jupyter/pull/487/commits/d0cfc979fcae2c073907f16957dc3b303127154b, or in `init.el` remove and re-add `org-babel-jupyter-make-local-aliases` to org-mode-hook with a positive `depth` value (i.e. add it later in the list).

(remove-hook 'org-mode-hook #'org-babel-jupyter-make-local-aliases)
(add-hook 'org-mode-hook #'org-babel-jupyter-make-local-aliases 10)

**Comment by nnicandro** [comment]
Should be fixed by #501.
